### PR TITLE
feat: add integration test to illustrate minimum set of hbase dependencies required for bigtable-hbase-2x

### PR DESCRIPTION
**Pull request by igorbernstein2**

### CODE REVIEW

**Review by mutianf — APPROVED** [review]

File: bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml [hunk 1]
```diff
@@ -36,9 +36,241 @@ limitations under the License.
...
+      </dependencies>
+    </profile>
+
+    <!-- NOTE: this profile is controlled via `int-test-1x-deps.mode` property -->
+    <profile>
+      <id>cbt-shaded-hbase-unshaded-deps</id>
+
+      <activation>
+        <property>
+          <name>int-test-2x-deps.mode</name>
```
Comment (mutianf) [inline bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml:89]: should this be int-test-1x-deps.mode? Or should the NOTE be int-test-2x-deps.mode?

File: bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml [hunk 2]
```diff
@@ -36,9 +36,241 @@ limitations under the License.
...
+      </dependencies>
+    </profile>
+
+    <!-- NOTE: this profile is controlled via `int-test-1x-deps.mode` property -->
+    <profile>
+      <id>cbt-shaded-hbase-minimal-unshaded-deps</id>
+
+      <activation>
+        <property>
+          <name>int-test-2x-deps.mode</name>
```
Comment (mutianf) [inline bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml:121]: same question here

**Review by mutianf — APPROVED** [review]
